src/renderer/components/opds/EntryPublicationList.tsx

```diff
@@ -16,12 +16,19 @@ import ListView from "readium-desktop/renderer/components/utils/ListView";
 import Loader from "readium-desktop/renderer/components/utils/Loader";
 import { RootState } from "readium-desktop/renderer/redux/states";
 
+import PageNavigation from "./PageNavigation";
+
 interface IProps extends RouteComponentProps {
     publications: OpdsPublicationView[];
+    goto: (url?: string) => void;
```

Comment:
I don't think `url` should be an optional (i.e. possibly `undefined`) function argument. In fact, callers of the `goto()` function ensure it is never called with a falsy value.